data/json/items/archery.json

```diff
@@ -571,7 +606,52 @@
       "dispersion" : 210,
       "durability" : 6,
       "clip_size" : 1,
-      "reload" : 70
+      "reload" : 70,
+      "valid_mod_locations": [["dampening", 1]]
+   },
+   {
+      "id" : "woodgreatbow",
+      "copy-from" : "longbow",
+      "override" : true,
+      "type" : "GUN",
+      "name" : "wooden greatbow",
+      "name_plural" : "wooden greatbows",
+      "description" : "An extremely large and stiff longbow, made with enormous limbs and a thick string to take an enormous amount of energy.  Takes incredible strength to draw.",
+      "price" : 150000,
+      "min_strength" : 18,
+      "weight" : 3200,
+      "volume" : 25,
+      "armor_data" : {
+         "covers" : ["TORSO"],
+         "coverage" : 7,
+         "material_thickness" : 2,
+         "encumbrance" : 20
+      },
+      "dispersion" : 200,
+      "durability" : 8,
+      "reload" : 100,
+      "loudness": 16,
+      "ranged_damage" : 35,
+      "range" : 12
+   },
+   {
+      "id" : "compgreatbow",
+      "copy-from" : "compbow",
+      "override" : true,
+      "type" : "GUN",
+      "name" : "compound greatbow",
+      "name_plural" : "compound greatbows",
+      "description" : "An extremely large and powerful compound bow, made with enormous limbs and a thick string to take an enormous amount of energy.  Takes incredible strength to draw.",
+      "price" : 160000,
+      "min_strength" : 18,
+      "weight" : 2000,
+      "volume" : 16,
+      "dispersion" : 100,
```

Comment:
Dispersion value of 100 is not changed compared to "compbow".

Reply:
Removed that line.